mod_wasm/modules/wasm/mod_wasm.h

```diff
@@ -5,5 +5,5 @@
 
 // mod_wasm version
 #define MOD_WASM_VERSION_MAJOR 0
-#define MOD_WASM_VERSION_MINOR 8
+#define MOD_WASM_VERSION_MINOR 10
```

Comment:
Why are we skipping 9? Just curious 🤔

Reply:
I was tempted to assign v1.0.0 since I believe we are very close to production-ready :-)
But I guess we can wait a little bit more. 

Then, given the major features incorporated in this new release, I thought 0.9.0 wouldn't have reflected such a change...